client/verta/verta/registry/check_model_dependencies.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Utility function for checking model dependencies against an environment
+to identify missing packages."""
+
+from typing import Set, Type
+import warnings
+
+from verta.registry import VertaModelBase
+from verta.environment import Python
+from verta._internal_utils import model_dependencies as md
+
+
+def check_model_dependencies(
+        model: Type[VertaModelBase],
+        environment: Python,
+        raise_for_missing: bool = False,
+        ) -> bool:
+    """Scan for missing or unused dependencies in a model's environment
+    to help avoid deployment failures and reduce time to deploy
+
+    This function attempts to scan the provided model class for 3rd-party (not
+    python standard library) dependencies and identify any missing or
+    extraneous packages in the provided environment.
+
+    .. note::
+        This function is not guaranteed to detect all dependencies in all cases.
+
+    .. versionadded:: 0.22.2
+
+    Parameters
+    ----------
+    model: subclass of :class:`~verta.registry.VertaModelBase`
+        Model class object (not an instance) to be scanned.
```

Comment:
I think the word "object" makes it a bit confusing; in most languages, "object" directly implies "instance" and I'm not sure how common the knowledge is that Python `class`es are objects.

```suggestion
        Model class (not an instance) to be scanned.
```

Reply:
fixed